### DISCUSSION TITLE
Activity Log UI

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15102] [Feature Requests]
I noticed the **Activity Log** provides both `x` and ✅  buttons.
It's a tiny UI thing, but perhaps we should modify this, since it is read-only.

<img width="1721" alt="Screen Shot 2022-08-16 at 5 20 58 PM" src="https://user-images.githubusercontent.com/62306630/184995969-62a117c8-158f-45f7-a754-8d923392f722.png">

**Comment by rijkvanzanten** [comment]
Good one, ✔️ doesn't do anything in this case

**Reply by erondpowell** [reply]
Okay. Started an issue from this.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
